subsys/sal/sid_pal/src/sid_crypto_keys.c

```diff
@@ -82,7 +82,7 @@ int sid_crypto_keys_new_import(psa_key_id_t id, uint8_t *data, size_t size)
 	/* Import key to secure storage */
 	status = psa_import_key(&attributes, data, size, &out_id);
 	if (PSA_SUCCESS == status && out_id == id) {
-		LOG_HEXDUMP_DBG(data, size, "found new key: ");
+		LOG_DBG("psa_import_key success");
```

Comment:
```suggestion
		LOG_DBG("psa_import_key success.");
```

Reply:
Agreed not to place dots and the end of comments